modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenOperation.java

```diff
@@ -207,6 +208,7 @@ public boolean getAllResponsesAreErrors() {
      * returns a map where the key is the request body content type and the value is the current CodegenOperation
      * this is needed by templates when a different signature is needed for each request body content type
      */
+    @JsonIgnore
```

Comment:
Why is this field even showing up? I don't see it in the toString method.

Reply:
That is correct, because it is not. The method is only used in the Python template, but it still somehow gets called by Jackson by reflection during serialization : 

<img width="1143" alt="image" src="https://user-images.githubusercontent.com/921666/207959133-e55254af-1a65-469d-a008-890d20697f8b.png">